open_lm/main.py

```diff
@@ -195,7 +196,18 @@ def save_checkpoint(
         if scaler is not None:
             checkpoint_dict_opt["scaler"] = scaler.state_dict()
 
-        if completed_epoch == args.epochs or (args.save_frequency > 0 and (completed_epoch % args.save_frequency) == 0):
+        checkpoint_dict_stats = {
+            "epoch": completed_epoch,
+            "name": args.name,
+            "is_final_checkpoint": is_final_checkpoint,
+            "evaluation_loss": evaluation_loss,
+        }
```

Comment:
We're not using these stats right now, right? We just create them for inspection?